tests/subsys/tracing/testcase.yaml

```diff
@@ -0,0 +1,17 @@
+tests:
+  tracing.transport:
+    platform_whitelist: native_posix
+    extra_args: CONF_FILE="prj.conf"
+    tags: tracing_disable
+  tracing.transport.uart:
+    platform_whitelist: qemu_x86
```

Comment:
Since it can run on qemu_x86, why can it not run on a physical board?

Reply:
Because we just use qemu to generate uart tracing data,  and handle the data in native platform(real case).